tests/tracer/test_logger.py

```diff
@@ -260,7 +260,7 @@ def test_logger_handle_debug(self, base_handle):
 
         # Assert that we did not perform any rate limiting
         total = 1000 * len(ALL_LEVEL_NAMES)
-        self.assertEqual(base_handle.call_count, total)
+        assert total <= base_handle.call_count <= total + 1
```

Comment:
nit: Should we be using `self.assertTrue(total <= base_handle.call_count <= total + 1)` for syntax consistency here?

Reply:
yeah, good call